app/lib/utils/getFilteredOutTermsQuery.js

```diff
@@ -1,18 +1,39 @@
 const constants = require('../constants');
 
 function getFilter(searchType) {
+  const mustNotClause = [
```

Comment:
As these are all constant settings, have you considered structuring this as a JSON dictionary, and having a lookup for each search type? i.e. 
```
const mustNotClause = [
  { match: { name: 'Marie Stopes' } },
  { match: { name: 'Young People Friendly Practice' } },
  { match: { name: 'BPAS' } },
];

const searchFilters = {
  sexual_health_professionals_geo: {
    mustClause: [
      { match: { type: 'Sexual health information and support' } },
      { match: { venueType: 'Clinic' } },
    ],
    mustNotClause,
  },
  ...
```

It might make it easier to see at a glance what the filters are on each service type.

Reply:
I've refactored the query a bit, but adding an extra key when we already have a label for the esQuery might be overkill (it shows which constant is related too)

```
esQuery: {
  "label": "sexual_health_professionals_geo",
  "query": {
    "body": {
      "query": {
        "bool": {
          "must": [
            {
              "match": {
                "type": "Sexual health information and support"
              }
            },
...
```